python/ray/data/_internal/arrow_block.py

```diff
@@ -608,7 +596,7 @@ def merge_sorted_blocks(
     @staticmethod
     def aggregate_combined_blocks(
         blocks: List[Block],
-        key: Union[str, List[str]],
+        sort_key: "SortKey",
```

Comment:
you should also update the top level typehint in `python/ray/data/block.py`

Reply:
updated